arrays.dd

```diff
@@ -346,16 +346,17 @@ a[] = b[] + 4;
 ---
 
         $(P A vector operation is indicated by the slice operator appearing
-        as the lvalue of an =, +=, -=, *=, /=, %=, ^=, $(AMP)= or |= operator.
-        The rvalue can be an expression consisting either of an array
-        slice of the same length and type as the lvalue or an expression
-        of the element type of the lvalue, in any combination.
+        as the left-hand side of an =, +=, -=, *=, /=, %=, ^=, $(AMP)= or |=
+        operator.
+        The right-hand side can be an expression consisting either of an array
+        slice of the same length and type as the left-hand side or an expression
+        of the element type of the left-hand side, in any combination.
         The operators supported for vector operations are the binary
         operators +, -, *, /, %, ^, $(AMP) and |, and the unary operators
         - and ~.
         )
 
-        $(P The lvalue slice and any rvalue slices must not overlap.
+        $(P The slice on the left and any slices on the right must not overlap.
```

Comment:
Any here reads like "at least one", which makes it "The slice on the left and at least one of the slices on the right must not overlap". I'm pretty sure that's not what is intended because it implies that some of the slices on the right **can** overlap.